CHANGELOG.md

```diff
@@ -0,0 +1,88 @@
+<a name="v1.1.0"></a>
+## v1.1.0
+
+### Build
+- **config:** update cors defaults
+- **deps:** bump [@types](https://github.com/types)/pg from 7.14.8 to 7.14.9
+- **deps:** bump ws from 7.4.1 to 7.4.2
+- **deps:** bump express-promise-router from 3.0.3 to 4.0.1
+- **deps:** bump [@types](https://github.com/types)/pg from 7.14.7 to 7.14.8
+- **deps-dev:** bump eslint from 7.17.0 to 7.18.0
+- **deps-dev:** bump [@typescript](https://github.com/typescript)-eslint/parser from 4.14.0 to 4.14.1
+- **deps-dev:** bump [@typescript](https://github.com/typescript)-eslint/eslint-plugin
+- **deps-dev:** bump sinon from 9.2.3 to 9.2.4
+- **deps-dev:** bump [@types](https://github.com/types)/node-forge from 0.9.6 to 0.9.7
+- **deps-dev:** bump [@types](https://github.com/types)/node from 14.14.21 to 14.14.22
+- **deps-dev:** bump eslint-config-standard-with-typescript
+- **deps-dev:** bump [@typescript](https://github.com/typescript)-eslint/eslint-plugin
+- **deps-dev:** bump [@typescript](https://github.com/typescript)-eslint/parser from 4.13.0 to 4.14.0
+- **deps-dev:** bump eslint from 7.18.0 to 7.19.0
+- **deps-dev:** bump [@types](https://github.com/types)/node from 14.14.20 to 14.14.21
+- **deps-dev:** bump [@typescript](https://github.com/typescript)-eslint/parser from 4.14.1 to 4.14.2
+- **deps-dev:** bump [@typescript](https://github.com/typescript)-eslint/eslint-plugin
+- **deps-dev:** bump [@typescript](https://github.com/typescript)-eslint/eslint-plugin
+- **deps-dev:** bump [@types](https://github.com/types)/ws from 6.0.4 to 7.4.0
+- **deps-dev:** bump [@types](https://github.com/types)/jest from 26.0.19 to 26.0.20
+- **deps-dev:** bump ts-jest from 26.4.4 to 26.5.0
+- **deps-dev:** bump sinon from 7.5.0 to 9.2.3
+- **deps-dev:** bump ts-node from 8.6.2 to 9.1.1
+- **deps-dev:** bump [@types](https://github.com/types)/node-vault from 0.9.1 to 0.9.13
+- **deps-dev:** bump jest-sinon from 1.0.2 to 1.0.4
+- **deps-dev:** bump nodemon from 2.0.3 to 2.0.7
+- **deps-dev:** bump [@typescript](https://github.com/typescript)-eslint/parser from 4.12.0 to 4.13.0
+
+### Ci
+- add types for conventional commits
+- add docker push to ci
+- add lint to build requirements
+- remove snyk from github actions
+
+### Docs
+- add changelog
+- add status badges
+- add release disclaimer
+- update readme for how to run API tests locally
+
+### Feat
+- **cors:** allow withCredentials to be sent to server
+- **cors:** add support for CORS
```

Comment:
Added network configuration on AMT device
Added a feature to update MEBx password on AMT device